docs/user/ovirt/install_ipi.md

```diff
@@ -108,8 +108,9 @@ Continue the installation using the install-config in the new folder `install_di
 $ openshift-install create cluster --dir=install_dir
 ``` 
 
-When the all prompts are done the installer will create 3 VMs under the
-Cluster supplied, and another VM as the bootstrap node.
+When the all prompts are done the installer will create ${HOME}/.ovirt/ovirt-config.yaml
```

Comment:
I'm not the best one to suggest docs modifications, but should I try to suggest the following:
When the all prompts are done the installer will create ${HOME}/.ovirt/ovirt-config.yaml containing all the required information about connection with the Engine.
Three VMs and a bootstrap node will be also created under the supplied Cluster.